totals.py

```diff
@@ -0,0 +1,24 @@
+"""
+Total income program
+"""
+
+def main():
+    """Display income report for incomes over a given number of months."""
+    incomes = []
+    months = int(input("How many months? "))
```

Comment:
This name seems plural, like it might be a list...

Reply:
Good point. 👈